<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>表格名称</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox_tableName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ProjectName">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>项目（软件）编号-项目（软件）名称</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="comboBox_ProjectName"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_TestNumber">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>测试用例标识</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBox_TestNumber"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_priority">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>优先级</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QComboBox" name="comboBox_priority"/>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_requirementNumber">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>软件需求/模块标识</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBox_requirementNumber"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_designer">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>用例设计人员</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_designer"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_designTime">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>设计时间</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="comboBox_designTime"/>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_requirementModel">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>软件需求/模块名称</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QComboBox" name="comboBox_requirementModel"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_TestName">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>用例说明</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="5">
           <widget class="QComboBox" name="comboBox_TestName"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_TestType">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>测试类型</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="5">
           <widget class="QComboBox" name="comboBox_TestType">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_condition">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>前提条件</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="5">
           <widget class="QComboBox" name="comboBox_condition">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_TestStep">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>测试步骤</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="5">
           <widget class="QComboBox" name="comboBox_TestStep">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_expected">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>预期输出</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="5">
           <widget class="QComboBox" name="comboBox_expected">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
